### PR TITLE
Logic Enhancement for DeleteEmployee

### PR DESCRIPTION
**Pull request by alfredkohhh**
Current Implementation:
- Deleting an Employee will not remove the deleted employee from the Task assigned list but is now deleted from ManageEZPZ.

New Implementation:
- Deleting an Employee will now untag the Employee from all Tasks that the Employee is being assigned to.
- After the above, the normal deletion of the Employee will take place.

### CODE REVIEW

File: src/main/java/manageezpz/logic/commands/DeleteEmployeeCommand.java [hunk 1]
```diff
@@ -33,13 +35,20 @@ public DeleteEmployeeCommand(Index targetIndex) {
...
 
-        if (targetIndex.getZeroBased() >= lastShownList.size()) {
+        if (targetIndex.getZeroBased() >= lastShownPersonList.size()) {
             throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
         }
 
-        Person personToDelete = lastShownList.get(targetIndex.getZeroBased());
+        Person personToDelete = lastShownPersonList.get(targetIndex.getZeroBased());
+        List<Task> taskList = lastShownTaskList.stream()
+                .filter(task -> task.getAssignees().contains(personToDelete)).collect(Collectors.toList());
```
Comment (alfredkohhh) [inline src/main/java/manageezpz/logic/commands/DeleteEmployeeCommand.java:47]: @Denniszedead Hey thanks for the recommendation !

**Review by Denniszedead — APPROVED** [review]
LGTM! 👍🏻

File: src/main/java/manageezpz/logic/commands/DeleteEmployeeCommand.java (hunk 1 above)
Comment (ChanWeiJie) [inline src/main/java/manageezpz/logic/commands/DeleteEmployeeCommand.java:47]: The code looks neat! Good job~

**Review by ChanWeiJie — APPROVED** [review]
LGTM!

**Pull request merged**